### PR TITLE
Added hardtanh activation function and test to PaddlePaddle frontend

### LINKED ISSUE #16189: hardtanh

### PR DESCRIPTION
**Pull request by sachelsout**
Close #16189

### CODE REVIEW

File: ivy/functional/frontends/paddle/nn/functional/activation.py [hunk 1]
```diff
@@ -31,3 +31,11 @@ def selu(
 
 tanh = paddle_tanh
 log_softmax = paddle_log_softmax
+
+
+@with_supported_dtypes({"2.4.2 and below": ("float32", "float64")}, "paddle")
+@to_ivy_arrays_and_back
+def hardtanh(x, min=-1.0, max=1.0):
```
Comment (Daniel4078) [inline ivy/functional/frontends/paddle/nn/functional/activation.py:38]: I think that you should mimik how the selu function above write its function signature, specifically how it specifes that some arguments that include name are optional keyword_only arguments.

**Review by Daniel4078 — CHANGES_REQUESTED** [review]
I think that you should mimik how the selu function above write its function signature, specifically how it specifes that some arguments that include name are optional keyword_only arguments.

**Comment by sachelsout** [comment]
> I think that you should mimik how the selu function above write its function signature, specifically how it specifes that some arguments that include name are optional keyword_only arguments.

Yes, I completely missed 'name' argument. I'll make changes right now. :)

**Review by Daniel4078 — APPROVED** [review]
looks good to me!

**Pull request merged**